mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/repositories/AvtaleRepository.kt

```diff
@@ -178,8 +178,12 @@ class AvtaleRepository(private val db: Database) {
         val order = when (filter.sortering) {
             "navn-ascending" -> "a.navn asc"
             "navn-descending" -> "a.navn desc"
-            "status-ascending" -> "a.avslutningsstatus asc, a.start_dato asc, a.slutt_dato desc"
-            "status-descending" -> "a.avslutningsstatus desc, a.slutt_dato asc, a.start_dato desc"
+            "leverandor-ascending" -> "a.leverandor_organisasjonsnummer asc"
+            "leverandor-descending" -> "a.leverandor_organisasjonsnummer desc"
```

Comment:
Sondre nevnte i et Trello-kort at sortering på leverandør blir veldig rar fordi denne vil bare sortere på verdien til organisasjonsnummeret. Så enn så lenge så tror jeg det blir rart å ha med sortering på leverandør. Litt på samme måte som for status.

Reply:
Jepp, vi snakka om det igår, og Sondre lagde noen nye Trello-kort for å løse problematikken. Jeg tenker også at vi kan ta bort sorteringen på leverandør og arrangør for tiltaksgjennomføringer frem til det er løst?

Reply:
Ja, få det bort